Cloudinary/Classes/Core/Features/Helpers/Results/Helpers/CommonResultKeys.swift

```diff
@@ -56,6 +56,7 @@ internal enum CommonResultKeys: CustomStringConvertible {
         case .qualityAnalysis:       return "quality_analysis"
         case .accessibilityAnalysis: return "accessibility_analysis"
         case .coordinates:           return "coordinates"
+        case .filenameOverride:      return "original_filename"
```

Comment:
Why `original_filename` is mapped to `.filenameOverride`?

Reply:
The key was correct, the case is a mistake, should be "originalFileName"